README.md

```diff
@@ -275,33 +302,35 @@ The Tool example will run on `http://localhost:3000/` in your browser.  The exam
 > 
 > If the URLs are not properly formatted and/or the GitHub URL doesn't launch successfully, the student will see an error message.  With a valid project, when the student clicks Submit, s/he will be shown the resulting grade and the Tool uses the Library to pass the grade back to the Platform.
 > 
-> Finally, when the student clicks Done, the student is returned to the Platform.  The Teacher or Administrator on the Platform should be able to see that a grade has been used for test student for the example Tool.
+> Finally, when the student clicks Done, the student is returned to the Platform.  The Teacher or Administrator on the Platform should be able to see that a grade has been used for test student for the Example Tool.
 
 
 ---
 
-#### Test Suite
+#### C. Test Suite
 
-The Tool example provides a test suite to verify portions of the basic functionality of the Library.  
+The Library provides a test suite to verify portions of the basic functionality.  
 
 The tests require the use of a .env file, which if you are new to Node, is a a simple configuration text file that is used to pass customized variables into your application’s environment. 
 
-You will need to load the test data in your .env file for the follwoing.  You can use LMS Global's Reference Implementation Tool to generate keys (https://lti-ri.imsglobal.org/keygen/index).
-* CLIENT_ID=client12345
-* CLIENT_SECRET=<RSA Private Key>
-* CLIENT_PUBLIC=<Public Key>
+You will need to load the test data in your .env file for the following:
```

Comment:
For example tool purposes, database storage should occur locally using 'mongodb://localhost:27017/TESTLTI' in the mongoose.connect() function. Can mention that for production, they should implement a cloud storage service. 

NO NEED FOR .env in example tool.

Reply:
removed